### DISCUSSION TITLE
blog/growth-strategies-for-ordinary-people-starting-from-scratch

### DISCUSSION THREAD

**Discussion by giscus** [bbruceyuan/bbruceyuan.github.io#16] [Announcements]
# blog/growth-strategies-for-ordinary-people-starting-from-scratch

国庆七天，开始在 B 站和 YouTube 做了一些教学视频，非常开心地涨粉 1.5k+，以下是我做的几个策略和思考：1. 视频大于文章；2. 利他优先；3. 拙劣的完成好过比完美的开始；4. 选取合适的平台；5. 内容互相链接...

https://bruceyuan.com/blog/growth-strategies-for-ordinary-people-starting-from-scratch.html

**Comment by DayuGuo** [comment]
真不错，加油！

**Reply by bbruceyuan** [reply]
哈哈哈，突然想到了五月份和你聊说被看见是一个很重要的事情，想开始尝试做点对别人有用的事情，同时对自己有意义🤣

**Reply by DayuGuo** [reply]
感兴趣的话，可以做做这个问卷！
https://anotherdayu.com/2024/5962/

**Reply by bbruceyuan** [reply]
最近周末空闲时间比较沉迷于录制视频，等找时间写一写🤣~~~

**Comment by ChenXinXuTUSC** [comment]
在 B 站看到了 UP 的 LLM 从零开始视频，非常不错，希望 UP 主以后能做更多 “从零开始” 系列，很喜欢这种跟着动手实践写 toy 的文章与视频，谢谢 UP 无偿的知识分享🥳

**Reply by bbruceyuan** [reply]
哈哈哈，有用就好，感谢你的鼓励❤️。不过还是工作要紧，周末有时间的话会多多更新的~

**Comment by linkmust** [comment]
太厉害了博主，从哔哩哔哩来的 哈哈哈

**Reply by bbruceyuan** [reply]
一直想写一个新的思考，但是还没时间写（😮‍💨）
